src/contracts/GPv2Settlement.sol

```diff
@@ -127,6 +136,13 @@ contract GPv2Settlement {
         revert("Final: not yet implemented");
     }
 
+    /// @dev Invalidate onchain an order that has been signed offline.
+    /// @param orderDigest The unique digest associated to the parameters of an
+    /// order. See [`orderId`] for details.
+    function deleteOrder(bytes32 orderDigest) public {
```

Comment:
naming-nit: I would call this `invalidateOrder` or `cancelOrder` as `delete` might get confused with storage deleting WRT order gas refund.